code/src/actors/king_zora.c

```diff
@@ -20,6 +20,9 @@ void EnKz_rUpdate(Actor* thisx, GlobalContext* globalCtx) {
     // Zora's Domain scene, the position check is just to add a 1 frame delay
     if (gSettingsContext.kingZoraSpeed != KINGZORASPEED_VANILLA && globalCtx->sceneNum == 88 && thisx->speedXZ != 0 && thisx->world.pos.x < 628.0f) {
         u8 mweepCount = (gSettingsContext.kingZoraSpeed)? Bias(Hash(0)) + 1 : 1;
+        if(gSettingsContext.kingZoraSpeed==KINGZORASPEED_CUSTOM){
+            mweepCount=gSettingsContext.exactZoraSpeed+1;
+        }
```

Comment:
```suggestion
        if (gSettingsContext.kingZoraSpeed == KINGZORASPEED_CUSTOM){
            mweepCount = gSettingsContext.exactZoraSpeed + 1;
        }
```
Could you change the spacing around the operators on these lines to match the surrounding style? Styling usually isn't too important to me, but for some reason this being different from the rest of the file really bothers me.

Reply:
Sure I guess, I honestly like this format better but yeah I can do that.

Reply:
Done